lib/rmg/TreasurePlacer.cpp

```diff
@@ -653,11 +655,10 @@ ObjectInfo * TreasurePlacer::getRandomObject(ui32 desiredValue, ui32 currentValu
 		int r = generator.rand.nextInt(1, total);
 		
 		//binary search = fastest
-		auto it = std::lower_bound(thresholds.begin(), thresholds.end(), r,
-								   [](const std::pair<ui32, ObjectInfo*> &rhs, const int lhs)->bool
-								   {
-			return (int)rhs.first < lhs;
-		});
+		auto it = std::lower_bound(thresholds.begin(),
```

Comment:
Weird formatting

Reply:
How this should be done right, it both prev and now formatting is weird?

Reply:
Redone it using auto helper variable, i think with helper variable it looks better.